stubs/pycurl/pycurl.pyi

```diff
@@ -39,7 +39,7 @@ class CurlMulti:
     def setopt(self, option: int, value: Incomplete) -> None: ...
     def perform(self) -> tuple[Incomplete, int]: ...
     def fdset(self) -> tuple[list[Incomplete], list[Incomplete], list[Incomplete]]: ...
-    def select(self, timeout: float = ...) -> int: ...
+    def select(self, timeout: float) -> int: ...
```

Comment:
The docstring is `select([timeout]) -> number of ready file descriptors or -1 on timeout` but it indeed doesn't appears to require the argument at runtime.